### PR TITLE
Regular user

### PR DESCRIPTION
**Pull request by stg101**

### CODE REVIEW

File: app/views/rooms/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+<% @bookings.each do |booking| %>
+  <% if @conflict_ids.include? booking.id %>
+    <li><strong> <%= booking.inspect %> : conflict</strong> </li> 
+  <% else %>
+    <li><%= booking.inspect %></li> 
+  <% end %> 
+<% end %>
+</ul>
+
+=======
```
Comment (cristianbgp) [inline app/views/rooms/show.html.erb]: Please solve this conflict

File: db/seeds.rb [hunk 2]
```diff
@@ -59,7 +59,8 @@ def get_image(file_name)
 # Add 3 bookings
 # Check relation booking with room and user
 p "Adding 3 bookings"
-book1 = user1.bookings.create!(start_date: Date.parse("May 8 2018"), end_date: Date.parse("May 10 2018"), paid_price: room1.price, room_id: 1)
+book1 = user1.bookings.create!(start_date: Date.today, end_date: Date.today + 1.days, paid_price: room1.price, room_id: 1)
```
Comment (CesarCachay) [inline db/seeds.rb]: Which is the main purpose of the change in the seeds?

File: db/seeds.rb [hunk 3]
```diff
@@ -76,3 +77,21 @@ def get_image(file_name)
 promotion3 = room5.promotions.create(start_date: Date.parse("May 15 2019"), end_date: Date.parse("June 15 2019"), discount_type: "Percentage", discount_amount: 9)
 # promotion1.room << room5
 
+
+
+# condition_1 = @room.bookings.where(
```
Comment (CesarCachay) [inline db/seeds.rb]: Do you need those lines? or you will delete it?

File: app/views/rooms/show.html.erb (hunk 1 above)
Comment (stg101) [reply app/views/rooms/show.html.erb]: Already solved

**Pull request merged**